packages/bolt/lib/bridge/bridge.ts

```diff
@@ -110,7 +110,7 @@ export async function bridgeBoltMessage(
 	if (type !== 'delete') {
 		for (const i of data) {
 			// since this key is used to prevent echo, 15 sec expiry should be enough
-			await bolt.redis?.set(`message-${i.id}`, JSON.stringify(data), {
+			await bolt.redis?.set(`message-temp-${i.id}`, JSON.stringify(data), {
```

Comment:
is there any reason for the addition of the `temp` part of this? i may be extremely stupid but from what i can tell this just complicates the code a bit more, which isn't exactly what i want to do unless there's reason to